### DISCUSSION TITLE
Welcome to Hackintoh-Ideapad-Gaming-3i Discussions!

### DISCUSSION THREAD

**Discussion by maycon2-0** [maycon2-0/Hackintoh-Ideapad-Gaming-3i#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by farhannovan** [comment]
Hi, what version of OC and what MacOS is it suitable for? Thanks

**Reply by maycon2-0** [reply]
0.7.5 only to Big Sur, I'll update de EFI to run Monterey.

**Reply by farhannovan** [reply]
> 0.7.5 only to Big Sur, I'll update de EFI to run Monterey.

can imessages and facetime login and run normally?

**Reply by farhannovan** [reply]
Hi, I have an issue, if it sleeps and wants to wake up the screen is blank. Is it the same with you? Thanks in advance

**Reply by maycon2-0** [reply]
Facetime and IMessage is working. But Sleeps and wake up is with inssue, I'm tring to fix, but I'm not a expert on hackintosh, I'm a beginner, I have made this post only for help because I can't find anyting about this model on web, if you found the solution for the sleep, please, help us.

**Reply by farhannovan** [reply]
I found this https://github.com/dragons3232/Lenovo-gaming-3-10750h-hackintosh-EFI

**Reply by maycon2-0** [reply]
The EFI folders are updated, I tested the old version and it didn't work, but can the new version work, need to try, thanks.

**Reply by maycon2-0** [reply]
I tried to take this EFI folder and adapt it to Monterey to fix Sleep, but without success, we will have to wait until it updates the EFI folder to Monterey.
